bot/internal/bot/label.go

```diff
@@ -73,6 +85,23 @@ func (b *Bot) labels(ctx context.Context, files []github.PullRequestFile) ([]str
 			continue
 		}
 
+		docs, _, err := classifyChanges(b.c.Environment, files)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+
+		if docs {
```

Comment:
I think this logic will run on any PR that contains docs changes adding backport labels. Is this intended? Should we run it on PRs that contain _only_ documentation, without any code changes?

Reply:
FWIW, I don't think we should merge this. The point of manually adding labels was to allow the author to specify when a backport is necessary and to which branches.

If we're going to just put the labels on all PRs, why not just try to automatically backport all PRs?

Reply:
@zmb3 I've been having the same reservations to be honest reviewing this. I'm not really sure I understand what problem this is trying to solve, this feels a little backwards tbh. From the description:

>  docs authors need to remember to add backport labels to their PRs so the automatic backport workflow can create branches

This is by design, no? You add backport label to backport your PR. If you forget, you backport manually. 

> If an author forgets to backport a PR, we need to wait until the next major release before it becomes visible in the default version of the docs site.

I'm not sure why do you need to wait? You can just backport and it will get deployed once the PR merges?

@ptgott @alexfortuno

Reply:
Just mentioning here for historical purposes, this discussion also happened in Slack and we responded there.